src/Bowling.js

```diff
@@ -0,0 +1,54 @@
+function Scorecard() {
+  this.score = 0;
```

Comment:
Is it worth adding underscores (eg. this._score) to mark these as private characteristics?